services/Directory/data/FilterListTag.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "filterListId": 2,
```

Comment:
```suggestion
    "filterListId": 1,
```

Reply:
I _think_ this is a typo. Agree, @Yuki2718?

Reply:
Yep, sorry for delayed fix. I missed notification.